packages/zoe/tools/manualTimer.js

```diff
@@ -186,7 +192,11 @@ export default function buildManualTimer(log, startValue = 0n, timeStep = 1n) {
           promiseKit.resolve(timestamp);
         },
       });
-      timer.setWakeup(delay, delayWaker);
+      // TODO Is this mismatch of absolute and relative a bug before this PR?
+      // https://github.com/Agoric/agoric-sdk/issues/5810
+      // I am temporarily extracing the bigint directly to avoid the type error,
+      // but it is clearly wrong.
+      timer.setWakeup(TimeMath.relValue(delay), delayWaker);
```

Comment:
It's a bug.  The code in vat-timer.js adds `delay` to `now`.   This gets through tests because many of them start the clock at zero.  

If tests pass, it would be useful to just fix it here.  
```suggestion
      timer.setWakeup(TimeMath.addAbsRel(ticks, delay), delayWaker);
```

Reply:
Done. Letting it run in CI to see if fixing the breaks anything. Resolving. Will unresolve if anything breaks.